packages/flutter/lib/src/rendering/object.dart

```diff
@@ -1232,6 +1244,18 @@ base class PipelineOwner with DiagnosticableTreeMixin {
 
   List<RenderObject> _nodesNeedingPaint = <RenderObject>[];
 
+  /// The [RenderObject]s which need to be painted in the next [flushPaint] pass.
+  ///
+  /// [RenderObject]s marked with [RenderObject.isRepaintBoundary] are added
+  /// when they are marked needing paint. Subclasses of [PipelineOwner] may use them
+  /// to invalidate caches or otherwise make performance optimizations.
+  /// Since nodes may be marked for layout at any time, they are best checked during
+  /// [flushPaint].
+  ///
+  /// Note that this does not include marked children of child [PipelineOwner]s.
```

Comment:
Another "note that" that should be rephrased.

Reply:
Done